spec/models/preserved_copy_spec.rb

```diff
@@ -234,5 +234,17 @@
     it 'returns no PreservedCopies with future timestamps' do
       expect(pcs_ordered_by_query).not_to include future_timestamp_pc
     end
+
+    context '.normalize_date(timestamp)' do
+      it 'given a String timestamp, returns a Time object' do
+        expect(described_class.send(:normalize_date, '2018-01-22T18:54:48')).to be_an_instance_of(Time)
+      end
+      it 'given a Time Object, returns the same Time object' do
+        expect(described_class.send(:normalize_date, Time.now.utc)).to be_an_instance_of(Time)
+      end
+      it 'given nil, returns the start of teh Epoch Time object' do
+        expect(described_class.send(:normalize_date, nil)).to be_an_instance_of(Time)
+      end
```

Comment:
Might we have a test for
- unparseable (as a date)
- day only (no time) -- show what happens (specific Time, not just Time object)
- month only (no day or time)
- year only

For "fun", if you want to see some ugly date strings, coming from our actual real production data, check out https://github.com/sul-dlss/stanford-mods/blob/master/spec/date_parsing_spec.rb.  Note that the cases I mention above are likely sufficient for exercising what might get typed into a rake task in this context ...